gaming-website/index.html

```diff
@@ -2,11 +2,64 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="stylesheet" href="src/index.css">
+    <link rel="stylesheet" href="src/index.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>GameWave</title>
   </head>
-  <body>
 
+  <body class="bg-black font-sans text-white">
+    <!-- Hero section -->
+    <div
+      class="h-screen w-full items-center bg-[url('/assets/home-page/hero.webp')] bg-cover bg-center bg-no-repeat"
```

Comment:
Even if setting the background like this is possible it should be avoided as it pollutes the html a lot.

Reply:
Sure thing, i believe i changed it in another PR but i'll double check :)